lib/log_blackbox.c

```diff
@@ -110,11 +110,11 @@ _blackbox_vlogger(int32_t target,
 	chunk += sizeof(uint32_t);
 
 	/* log message */
-	msg_len = qb_vsnprintf_serialize(chunk, QB_LOG_MAX_LEN, cs->format, ap);
-	if (msg_len >= QB_LOG_MAX_LEN) {
+	msg_len = qb_vsnprintf_serialize(chunk, max_size, cs->format, ap);
+	if (msg_len >= max_size) {
 	    chunk = msg_len_pt + sizeof(uint32_t); /* Reset */
 
-	    msg_len = qb_vsnprintf_serialize(chunk, QB_LOG_MAX_LEN,
+	    msg_len = qb_vsnprintf_serialize(chunk, max_size,
 		"Log message too long to be stored in the blackbox.  "\
 		"Maximum is QB_LOG_MAX_LEN" , ap);
```

Comment:
Not changing the message indicates me there are likely other issues to be hit,
at least in blackbox context.